docs/setup_instructions/windows_installation_instructions.md

```diff
@@ -180,7 +181,7 @@ If you need to install any of the required packages, use the function `install.p
 install.packages("optparse")
 
 # Install all necessary packages at once:
-install.packages(c("optparse", "renv", "rmarkdown", "tidyverse"))
+install.packages(c("tidyverse", "rmarkdown", "optparse", "renv", "here"))
```

Comment:
I would also make these and the packages on line 259 alphabetical if you do that with the ones above.